sdk/search/search/src/searchServiceClient.ts

```diff
@@ -144,6 +163,50 @@ export class SearchServiceClient {
     }
   }
 
+  /**
+   * Retrieves a list of existing Skillsets in the service.
+   * @param options Options to the list Skillsets operation.
+   */
+  public async listSkillsets(options: ListSkillsetsOptions = {}): Promise<Skillset[]> {
```

Comment:
Can you add a code sample for `listSkillsets` and `createSkillset` to the README? I forgot to add some for index.

Reply:
Done.